src/main/java/org/breedinginsight/services/impl/BreedingMethodServiceImpl.java

```diff
@@ -111,10 +122,64 @@ public void deleteBreedingMethod(UUID programId, UUID breedingMethodId) throws A
         dsl.transaction(() -> breedingMethodDAO.deleteProgramMethod(programId, breedingMethodId));
     }
 
-    private boolean validateBreedingMethod(ProgramBreedingMethodEntity method) {
-        return StringUtils.isNotBlank(method.getName())
-                && StringUtils.isNotBlank(method.getAbbreviation())
-                && StringUtils.isNotBlank(method.getCategory())
-                && StringUtils.isNotBlank(method.getGeneticDiversity());
+     boolean isMissingRequiredFields(ProgramBreedingMethodEntity method) {
```

Comment:
Please add the `private` modifier to this method

Reply:
Explicitly declared the method 'protected'.  Added comment `'protected' instead of 'private' so it is accessible to unit test.`